pkg/component/component.go

```diff
@@ -348,14 +347,17 @@ func GetComponent(client kclient.ClientInterface, componentName string, applicat
 // getRemoteComponentMetadata provides component metadata from the cluster
 func getRemoteComponentMetadata(client kclient.ClientInterface, componentName string, applicationName string, getUrls, getStorage bool) (Component, error) {
 	fromCluster, err := GetPushedComponent(client, componentName, applicationName)
-	if err != nil || fromCluster == nil {
-		return Component{}, errors.Wrapf(err, "unable to get remote metadata for %s component", componentName)
+	if err != nil {
+		return Component{}, fmt.Errorf("unable to get remote metadata for %s component: %w", componentName, err)
+	}
+	if fromCluster == nil {
+		return Component{}, nil
```

Comment:
I don't think it changes behaviour: errors.Wrap(err, string) is special as it returns nil if err is nil.

Reply:
Ah okay. I thought it would return a new error with the message specified.
Indeed, you are right. It is stated in the [documentation](https://pkg.go.dev/github.com/pkg/errors#Wrap) that:
> If err is nil, Wrap returns nil.